File: src/main/java/ru/exordia/spleef/game/commands/SpleefCommand.java
package ru.exordia.spleef.game.commands;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

import java.util.HashMap;
import java.util.Map;

public class SpleefCommand implements CommandExecutor {
    private final Map<String, Commands> commands = new HashMap<>();

    public void addCommands() {
        commands.put("item", new Item());
        commands.put("createarena", new CreateArena());
        commands.put("join", new Join());
        commands.put("leave", new Leave());
        commands.put("setwin", new SetWin());
        commands.put("setlose", new SetLose());
        commands.put("setspawn", new SetSpawn());
        commands.put("resetarena", new ResetArena());
    }

    @Override public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
        Player player = (Player) sender;
        addCommands();

        if (args.length < 1) {
            player.sendMessage(helpMessage());
        } else {
            String cmd = args[0];
            if (commands.get(cmd) != null) {
                commands.get(cmd).onCommands(player, args);
            }
        }

        return true;
    }

    public String helpMessage() {
        StringBuilder result = new StringBuilder();

        result.append("§7=========================================").append("\n");
        result.append("§f/spleef leave - §cВыйти с игры").append("\n");
        result.append("§f/spleef join - §cПрисоеденится к игре").append("\n");
        result.append("§f/spleef item - §cЭлемент для создания арены§7§l§o*").append("\n");
        result.append("§f/spleef resetarena - §cЗаполнить арену§7§l§o*").append("\n");
        result.append("§7=========================================");


        return result.toString();
    }
}

File: src/main/java/ru/exordia/spleef/game/commands/SpleefCommand.kt
package ru.exordia.spleef.game.commands

import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player
import java.util.HashMap
import java.util.Map

class SpleefCommand : CommandExecutor {
    private val commands: Map<String?, Commands?> = HashMap()
    fun addCommands() {
        commands.put("item", Item())
        commands.put("createarena", CreateArena())
        commands.put("join", Join())
        commands.put("leave", Leave())
        commands.put("setwin", SetWin())
        commands.put("setlose", SetLose())
        commands.put("setspawn", SetSpawn())
        commands.put("resetarena", ResetArena())
    }

    @Override
    fun onCommand(sender: CommandSender, command: Command?, label: String?, args: Array<String?>): Boolean {
        val player: Player = sender as Player
        addCommands()
        if (args.size < 1) {
            player.sendMessage(helpMessage())
        } else {
            val cmd = args[0]
            if (commands[cmd] != null) {
                commands[cmd].onCommands(player, args)
            }
        }
        return true
    }

    fun helpMessage(): String {
        val result = StringBuilder()
        result.append("§7=========================================").append("\n")
        result.append("§f/spleef leave - §cВыйти с игры").append("\n")
        result.append("§f/spleef join - §cПрисоеденится к игре").append("\n")
        result.append("§f/spleef item - §cЭлемент для создания арены§7§l§o*").append("\n")
        result.append("§f/spleef resetarena - §cЗаполнить арену§7§l§o*").append("\n")
        result.append("§7=========================================")
        return result.toString()
    }
}